pages/docs/manual/latest/installation.mdx

```diff
@@ -30,9 +30,14 @@ During development, instead of running `npm run build` each time to compile, use
 If you already have a JavaScript project into which you'd like to add ReScript:
 
 - Install ReScript locally as a [devDependency](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file):
-  ```sh
+  ```shhttps://github.com/coleea/rescript-lang.org
   npm install rescript --save-dev 
   ```
+  
+  If you use M1-based Mac Device, error might be occur. If so, try installing `npm@6.14` via `npm i -g npm@6.14`, then try again `npm install rescript --save-dev`.
```

Comment:
This note should probably be part of the `Prerequisite` headline on the very top. This important m1 note gets missed otherwise?

Reply:
Yes. I also think that Prerequisite section is better suitable.
Also, I found the case that installing npm@6.14  could not solve the error. In this case, the problem solved when using rosetta. So I suggest that some link for Rosetta tutorial might helpful for people. for example
https://derflounder.wordpress.com/2020/11/17/installing-rosetta-2-on-apple-silicon-macs/

by the way, should I do pull-request again after re-writing the document file or Is it ok to do nothing ? I ask for your understanding about git-novice (maybe silly question).
thank you

Reply:
@coleea If you fix the issues and `git push` again, it will be automatically reflected in this PR page.